src/utils/displayFunctions.ts

```diff
@@ -21,8 +21,10 @@ const getLogoForBrandPetname = (brandPetname: string) => {
   }
 };
 
-export const displayPetname = (pn: Array<string> | string) =>
-  Array.isArray(pn) ? pn.join('.') : pn;
+// XXX We remove the "Ibc" prefix so e.g. "IbcATOM" becomes "ATOM". Should
+// this be mapped out more explicitly?
```

Comment:
good question. yeah I think it would be safer to do this only for known transformations. It could would also be more clear when searching the code.

Reply:
Done